script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -182,7 +182,7 @@ But I could see it.
 And it scared me so much, I couldn't even walk.
 <Page51>
 Speaking of which, my ears had been frozen since then, and it had gotten quieter despite the noise.
-Only the wound on my chest reverberated like a siren.
+Only the wound in my chest reverberated like a siren.
```

Comment:
"Wound **in**" sounds more severe, and seems more popular
![2021-12-02-002428_2231x1016_scrot](https://user-images.githubusercontent.com/2476799/144362864-97145190-b9dd-4d58-9f87-7571dc317d56.png)